Tests/FuzzilliTests/ProgramBuilderTest.swift

```diff
@@ -313,7 +330,9 @@ class ProgramBuilderTests: XCTestCase {
         b.defineClass(withSuperclass: superclass) { cls in
             cls.defineConstructor(withParameters: [.string]) { _ in
                 let v0 = b.loadInt(42)
-                let v1 = b.createObject(with: ["foo": v0])
+                let v1 = b.createObject{ prop in
```

Comment:
I kind of feel like we should have some convenience constructors since this "create object with normal properties" is very common. Can we just create an overload of `createObject` in the ProgramBuilder with the old signature?

Reply:
Can do, thinking along the same lines, should we also have multiple CodeGenerators as well (e.g. generate an object with no props, objects with just data props, objects will data props +getters/setters)? Currently it's all just under one generator.

Reply:
I think I'm fine with one generator that can generate everything :)